### PR TITLE
cli: remove list machines cmd

### PR DESCRIPTION
**Pull request by jmikovic**
[#123](https://trello.com/c/AlY9vpyt/123-remove-the-list-machines-command-from-cli)
list-machine cmd no longer exists and libvirt is not used

### CODE REVIEW

**Comment by vinzenz** [comment]
Jenkins: ok to test

File: src/leappto/cli.py [hunk 1]
```diff
@@ -614,10 +598,7 @@ def print_migrate_info(text):
     elif parsed.action == 'check-target':
         target = parsed.target
 
-        lmp = LibvirtMachineProvider()
-        machines = lmp.get_machines()
-
-        machine_dst = _find_machine(machines, target)
+        machine_dst = _inspect_machine(target)
```
Comment (vinzenz) [inline src/leappto/cli.py]: Seeing this I think we need to have the custom user support here - That however would be a different PR

File: src/leappto/cli.py [hunk 2]
```diff
@@ -637,10 +618,7 @@ def print_migrate_info(text):
     elif parsed.action == 'destroy-container':
         target = parsed.target
 
-        lmp = LibvirtMachineProvider()
-        machines = lmp.get_machines()
-
-        machine_dst = _find_machine(machines, target)
+        machine_dst = _inspect_machine(target)
```
Comment (vinzenz) [inline src/leappto/cli.py]: Same here for the custom user support

**Comment by GAZDOWN** [comment]
I am also missing the documentation update. We should have it removed form leapp-tool page.

**Comment by vinzenz** [comment]
Taken over by #267
